src/geohash_helper.c

```diff
@@ -296,7 +296,7 @@ double geohashGetDistance(double lon1d, double lat1d, double lon2d, double lat2d
     v = sin((lon2r - lon1r) / 2);
     /* Reflects about 6nm on earth for comparing longitudes. */
     const double GEO_EPSILON = 1e-15;
-    /* if v == 0, or practically 0, we can avoid doing expensive math when lons are practically the same */
+    /* if v == 0, or practically 0, we can avoid doing expensive math when longitudes are practically the same */
```

Comment:
I understand `lon`+`s`, but it was the only use in the repository, `longitudes` is used in this file, and there didn't seem to be any good reason to shorten it....